deploy/webhook/webhook.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: admissionregistration.k8s.io/v1beta1
+kind: ValidatingWebhookConfiguration
+metadata:
+  name: spokeclustervalidators.admission.cluster.open-cluster-management.io
+webhooks:
+- name: spokeclustervalidators.admission.cluster.open-cluster-management.io
+  clientConfig:
+    service:
+      # reach the webhook via the registered aggregated API
+      namespace: default
+      name: kubernetes
+      path: /apis/admission.cluster.open-cluster-management.io/v1/spokeclustervalidators
+  rules:
+  - operations:
+    - CREATE
+    - UPDATE
+    apiGroups:
+    - cluster.open-cluster-management.io
+    apiVersions:
+    - v1
+    resources:
+    - spokeclusters
+  failurePolicy: Fail
```

Comment:
add `timeoutSeconds` with a value of 2-3 sec

Reply:
added